docs/essentials/introduction.md

```diff
@@ -11,6 +11,30 @@ A major strength of Storybook are [addons](/addons/) that extend Storybook’s U
 - [Backgrounds](./backgrounds.md)
 - [Toolbars & globals](./toolbars-and-globals.md)
 
+### Installing
+If you created your project with `sb init` since Storybook 6, this is already installed for you and you can skip the rest of this section.
+
```

Comment:
@maslade we could a little bit better regarding the wording as it reads a bit off. We could probably go with something like:

```md
### installation

If you're running `sb init` to add Storybook to your project, the essentials package (`@storybook/addon-essentials`) is already installed and configured for you . You can skip the rest of this section.

If you're upgrading from a previous Storybook version, you'll need to run the following command in your terminal:
....
```

Let me know what do you think and we'll move from there.

Reply:
That's much better -- appreciate the suggestion. I've updated the PR.

Reply:
@maslade something is off. I'm still seeing the same content as initial. I've checked the commits and there's only one and that one's content is before the suggestion i made in the comment above? Could you check on your end and see what's wrong?

Reply:
Bad push -- please check again.